### ISSUE TITLE
welcome

### ISSUE THREAD

**Issue by fangyaota** [HZZcode/Micrologic#8]
bro didnt follow me

**Comment by fangyaota** [comment]
Addittionally, it seems that bro has a low code level :(

**Comment by HZZcode** [comment]
well i dont understand your issue very well.
you mean i need comments for my C++ code, or i need documents for Micrologic language?

**Comment by HZZcode** [comment]
Plus, i just seen this project as a tool so i won't care about code quality xd

**Comment by fangyaota** [comment]
yes i mean documents😈 if any comments are better. sorry buddy i have bad english😚

**Comment by fangyaota** [comment]
anyway, would u like to add gui with qc? that might be a bit harder😘
